lib/assets/bower_components/bootstrap-toggle/.bower.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
I'm pretty positive we don't use bower anymore, and it was causing an issue for some reason (not really sure why).  Seemed like a good time to kill this folder.